jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/Dispatcher.java

```diff
@@ -664,6 +664,24 @@ public void sendRedirect(String location) throws IOException
         {
             // NOOP for include.
         }
+
+        @Override
+        public void sendRedirect(String location, boolean clearBuffer) throws IOException
+        {
+            // NOOP for include.
```

Comment:
Would there be any benefit to have these be LOG.debug() statements that indicate the skip?  That way someone that is boggled by this change can see the output in the debug logs? vs having the source code of Jetty to know what is going on?

Reply:
This is not really a change in behaviour, at least not against jetty-9/10/11 as they all noop methods on include.  So we are really just extending to a few more methods and fixing cross context.  Anybody who is boggled should read the javadoc of sendRedirect and it clearly says that it is a noop for include.
So I'd rather avoid the noise of lots of debugs.